src/main/java/ru/l1ttleO/boyfriend/Utils.java

```diff
@@ -55,16 +57,41 @@ public static String getBeep() {
         return out.toString();
     }
 
-    public static int getDurationMultiplied(final @NotNull String toParse) {
+    public static int getDurationMultiplied(final @NotNull String toParse) throws NumberFormatException {
         try {
+            return Integer.parseInt(toParse);
+        } catch (final @NotNull NumberFormatException ignored) {
+        }
+
+        int result = 0;
+        final Set<String> used = Collections.emptySet();
```

Comment:
```suggestion
        final Set<String> used = new java.util.HashSet<>(Collections.emptySet());
```
Reason: Fixes `Immutable object is modified: line 74` and `Condition 'used.contains(buffer[1])' is always 'false' when reached: line 72`